### DISCUSSION TITLE
Get all the items from a collection preset

### DISCUSSION THREAD

**Discussion by oswaldB** [directus/directus#3270] [Feature Requests]
Hello,

It would be awesome if it were possible to get all the items from a preset with one request in the api.

As you have an interface for non-technical users, they can create presets with specific items and so changing the items display in an application front-end without changing the api request.

;)

**Comment by benhaynes** [comment]
Love it! That's a great idea... 👍

**Comment by rijkvanzanten** [comment]
Heya! Thank you for taking the time to submit this request! 

It has been over **90 days**, and this discussion has not received at least **15 votes** from the community. This means that we don't feel like there's enough community interest to warrant further R&D into this topic at this time. :ice_cube:

This request will now be closed to keep our discussions tidy. Please reach out if you have any questions!

_For more information, see our [Feature Request Process](https://docs.directus.io/contributing/feature-request-process.html)._
